packages/babel-parser/ast/spec.md

```diff
@@ -718,6 +720,26 @@ interface ObjectMethod <: ObjectMember, Function {
 }
 ```
 
+## RecordExpression
+
+```js
+interface RecordExpression <: Expression {
+  type: "RecordExpression";
+  syntaxType: string;
+  properties: [ ObjectProperty | ObjectMethod | SpreadElement ];
```

Comment:
🤔Is `ObjectMethod` allowed in `RecordExpression`?

Reply:
Good question! My initial attempt disallowed `ObjectMethod` inside a `RecordExpression` but after the discussion on https://github.com/tc39/proposal-record-tuple/issues/88 I opted for the current behavior.